sonar-php-plugin/src/main/java/org/sonar/plugins/php/PHPSensor.java

```diff
@@ -140,13 +139,10 @@ private static boolean inSonarQube(SensorContext context) {
   }
 
   private void processCoverage(Map<String, Integer> numberOfLinesOfCode, SensorContext context) {
-    PhpUnitService phpUnitSensor = new PhpUnitService(
-      fileSystem,
-      new PhpUnitTestResultImporter(),
-      new PhpUnitCoverageResultImporter(fileSystem),
-      new PhpUnitItCoverageResultImporter(fileSystem),
-      new PhpUnitOverallCoverageResultImporter(fileSystem));
-    phpUnitSensor.execute(context, numberOfLinesOfCode);
+    new PhpUnitTestResultImporter().parseReport(context, numberOfLinesOfCode);
```

Comment:
@inverno I think we should rename `parseReport` (not much sense from this level of abstraction). Maybe even we can have different names for test results and coverage: `importTestResults` (btw `numberOfLinesOfCode` are not required there, it's confusing to see them as argument) and `importCoverageResults`